articles/100_ament.md

```diff
@@ -20,34 +20,93 @@ author: '[Dirk Thomas](https://github.com/dirk-thomas)'
 Original Author: {{ page.author }}
 
 
+## Preface
+
+ROS is developed with a federated model.
+While this comes with a lot of advantages it makes building a set of interdependent packages more complex.
+
+Commonly a user has to build interdependent packages in the correct topological order.
+And for each package the steps include reading the documentation of a package, then building and installing it following a few build system specific commands and then setting up the environment for other packages to use it.
+For a larger number of packages this approach does not scale well.
+
+ROS mitigates that complexity with automation in the build process.
+Wherever possible ROS uses existing, standard tools, e.g. CMake and Python setuptools.
+Where these tools lack built-in support for what ROS needs to do, the additional functionality is provided by ROS.
+
+
+### Origin of ament
+
+ament is an evolution of [catkin](http://wiki.ros.org/catkin).
+The word *ament* is actually a synonym of *catkin*.
+For more information about the differences please read [below](#how-is-ament-different-from-catkin).
+
+
 ## ament
 
-`ament` is a meta build system to build applications which are split into separate packages.
+ament is a meta build system to improve building applications which are split into separate packages.
+It consists of two major parts:
 
-It relies on meta information about the packages to determine their build dependencies.
-This meta information is specified in a file called `package.xml` which is defined in [REP 140](http://www.ros.org/reps/rep-0140.html).
+* the *build system* to configure, build, and install a single package
```

Comment:
Should this be `a *build system*`? Currently we have `ament_cmake` and `ament_python`, etc. Or by `*build system*` do you mean all build types? If you switch to `a *build system*` it would be good to include ` (e.g. ament_cmake, cmake, setuptools, etc.)`.